google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/testing/LocalPubSubHelperTest.java

```diff
@@ -50,25 +46,10 @@ public void testCreate() {
   }
 
   @Test
-  public void testOptions() {
-    LocalPubSubHelper helper = LocalPubSubHelper.create();
-    PubSubOptions options = helper.getOptions();
-    assertTrue(options.getProjectId().startsWith(PROJECT_ID_PREFIX));
-    assertTrue(options.getHost().startsWith("localhost:"));
-    assertSame(NoCredentials.getInstance(), options.getCredentials());
-  }
-
-  @Test
   public void testStartStopReset() throws IOException, InterruptedException, TimeoutException {
     LocalPubSubHelper helper = LocalPubSubHelper.create();
```

Comment:
I think the class can be deleted for reasons we already talked about. On the other hand, according to the doc, it's not only there to help test the client lib but also for any user who want to test their app against.

Up to you? I can be swayed both ways.